webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -483,7 +536,134 @@
 
                   rdfs:subPropertyOf knora-base:hasValue .
 
+    ### ###########################################
+    ### beol:hasFloruitDate
+
+    :hasFloruitDate rdf:type owl:ObjectProperty ;
+
+                  rdfs:label "Blühtedataum"@de ,
+                             "Floruit Date"@en ;
+
+                  rdfs:comment """Repräsentiert ein Blühtedataum."""@de,
```

Comment:
Ok, could you then just choose another label (especially in German) or explain it a little bit in the `rdfs:comment`?

In German, this sounds like the person is a flower :-)